### PR TITLE
Redo bisect pipeline

### PR DESCRIPTION
**Pull request by ader1990**

### CODE REVIEW

File: linux_pipeline/Jenkinsfile_kernel_bisect_commit_validator [hunk 1]
```diff
@@ -1,435 +1,669 @@
...
+}
+
+def prepareEnv(branch, remote) {
+    cleanWs()
+    git branch: branch, url: remote
+}
 
-perfServer = GetPerfServer(PERF_TEST)
-perfTestType = GetPerfTestType(PERF_TEST)
+def unstashKernel(kernelStash) {
```
Comment (mbivolan) [inline linux_pipeline/Jenkinsfile_kernel_bisect_commit_validator]: i think is a better idea to add another parameter to specify what to keep, and delete everything else

File: linux_pipeline/Jenkinsfile_kernel_bisect_commit_validator [hunk 2]
```diff
@@ -1,435 +1,669 @@
...
+        customKernel = "-CustomKernel 'localfile:./${env.BUILD_NUMBER}-${env.BRANCH_NAME}-${env.KERNEL_ARTIFACTS_PATH}/*/deb/*.deb'"
+    } catch (Exception e) {
+        echo "No stash ${env.KERNEL_ARTIFACTS_PATH} found"
     }
-}
 
-if (isPerf) {
-    node (perfServer) {
-        perf()
+    if (testName.equals(env.LISAV2_AZURE_STRESS_TEST)) {
```
Comment (mbivolan) [inline linux_pipeline/Jenkinsfile_kernel_bisect_commit_validator]: maybe add a parameter in this method for the vmsize, and check if its null or not

File: linux_pipeline/Jenkinsfile_kernel_bisect_commit_validator (hunk 1 above)
Comment (ader1990) [reply linux_pipeline/Jenkinsfile_kernel_bisect_commit_validator]: will keep in mind and I will do it when I refactor the developer/ bisect slave pipelines in one.

File: linux_pipeline/Jenkinsfile_kernel_bisect_commit_validator (hunk 2 above)
Comment (ader1990) [reply linux_pipeline/Jenkinsfile_kernel_bisect_commit_validator]: this runLisa method became too big and has to many params, will have to break it down somehow

**Pull request merged**